### PR TITLE
mostly fixed June Discord update messing up all the classes

### PR DESCRIPTION
**Pull request by Xanazf**
though i'm not sure if the rest of the code requires any fixes

sorry for formatting stuff, it's my ESLint + Prettier, i really only changed the class definitions under // Classes and relevant things

had to change the toolBar prop, split it into toolBarTop and toolBarMsg so that it could find the "more" icon (though i'm not sure why it doesn't work, so currently it just searches the entire document)

anyways, i hope this'll help adjusting the code to the new changes

### CODE REVIEW

File: CollapsibleUI.plugin.js [hunk 1]
```diff
@@ -212,333 +251,382 @@ module.exports = (() => {
...
+        // it appears to have an id of e44302
+        this.classSelected = "selected_e44302";
+        // assumed the icon bar to the right of search
+        this.classIconWrapper = "iconWrapper_e44302";
+        // assumed the most often used class
+        this.classClickable = "clickable_e44302";
+        // assumed the most often used class for icons
+        this.classIcon = "icon_e44302";
+        // there's also container?
+        this.classCallContainer = "wrapper_adcaac";
```
Comment (Aluerie) [inline CollapsibleUI.plugin.js:511]: I think this one is `this.classCallContainer = 'wrapper_d880dc';`

File: CollapsibleUI.plugin.js [hunk 2]
```diff
@@ -212,333 +251,382 @@ module.exports = (() => {
...
+        // assumed the most relevant to the name
+        this.classCallUserWrapper = "voiceUser_cdc675";
```
Comment (Aluerie) [inline CollapsibleUI.plugin.js:513]: this is probably `voiceCallWrapper_bae578`

Comment (Xanazf) [reply CollapsibleUI.plugin.js:513]: oh, right, there's also the big screen for the voice chat, i forgot about that, thanks

yeah i did hand-pick them, never really done BetterDiscord plugins or themes, but CollapsibleUI is just too convenient of a tool so when Discord did the thing i just had to get CollapsibleUI back ASAP, wasn't aware of the ecosystem

that's an awesome tool, though it missed a couple classes, definitely a better starting point than Electron DevTools (not Wayland yet)

**Comment by programmer2514** [comment]
Unfortunately, this is a bit difficult to review due to the linting tools messing with the diff. I have implemented ESLint going forward (without Prettier because I am not a fan of how many unnecessary line-breaks it introduces), so future pull requests should be easier to work with. I appreciate the contribution and will likely cross-reference this in the fix I am currently working on.

**Pull request closed without merging**